client/app/bundles/course/assessment/submission/translations.ts

```diff
@@ -323,6 +323,12 @@ const translations = defineMessages({
     id: 'course.assessment.submission.updateFailure',
     defaultMessage: 'Submission update failed: {errors}',
   },
+  startTimedExamAssessmentFailed: {
+    id: 'course.assessment.submission.startTimedExamAssessmentFailed',
+    defaultMessage:
+      'There is an error in starting the exam / assessment. Please check your \
```

Comment:
missing trailing full stop, and don't need to overload with both terminology "exam / assessment", can keep it short.

`There was an error starting the assessment. Please check your internet connection and try again.`